setup.cfg

```diff
@@ -0,0 +1,18 @@
+[flake8]
+
+#displays error count at the end of output
```

Comment:
Comments should be "# Display ... output." I.e. Should be full sentences with a leading "#" and "space"